File: 6_Security/Affine_Cipher/Encryption.java

public class Encryption {
 public String Encryption(String key, int a, int b, int a_inverse, int LP4N, String content) {
  String ciphertext = "";
  String ASCcode = AC.strToInt(content);
  String[] Int_ASC = ACScode.split("");

  for (int i = 0; i < Int_ACS.length; i++) {
   int temp = Integer.parseInt(Int_ASC[i]) * a + b;
   ciphertext = ciphertext + temp + "";
  }
  return ciphertext;
 }
}

File: 6_Security/Affine_Cipher/Encryption.py
#!/usr/bin/env python

def Encryption(key, a, b, a_inverse, LP4N, content):
    ciphertext = ""
    ASCcode = strToInt(content)
    Int_ASC = ASCcode.split("")
    i = 0
    while i < Int_ASC.length:
        temp = int(Int_ASC[i]) * a + b
        ciphertext = ciphertext + temp + ""
        i += 1
    return ciphertext